src/SparkFun_u-blox_GNSS_Arduino_Library.h

```diff
@@ -560,6 +562,8 @@ class SFE_UBLOX_GNSS
 	boolean begin(TwoWire &wirePort = Wire, uint8_t deviceAddress = 0x42); //Returns true if module is detected
 	//serialPort needs to be perviously initialized to correct baud rate
 	boolean begin(Stream &serialPort); //Returns true if module is detected
+	//SPI - supply instance of SPIClass, slave select pin and SPI speed (in Hz)
+	boolean begin(SPIClass &spiPort, uint8_t ssPin, int spiSpeed);
```

Comment:
Please change all references to ``ssPin``` to ```csPin```

Reply:
Done